### ISSUE TITLE
[Question] - Do you have the raw html (non-angular) version?

### ISSUE THREAD

**Issue by edwinyzh** [akveo/blur-admin#4]
Hi,

This is a very very nice looking theme, do you have the html version (non-angular) version? Thanks!

**Comment by KostyaDanovsky** [comment]
Hi @edwinyzh, unfortunately no, but if a lot of people want vanilla.js version we'll do it with pleasure.

**Comment by edwinyzh** [comment]
Hi Konstantin,

Actually I'm interesting using it as a template for creating a website, as
opposed a webapp, so I'm interested in the raw html files that looks like
http://akveo.com/blur-admin

:)

On Sun, Apr 10, 2016 at 4:57 PM, Konstantin Danovsky <
notifications@github.com> wrote:

> Hi @edwinyzh <https://github.com/edwinyzh>, unfortunately no, but if a
> lot of people want vanilla.js version we'll do it with pleasure.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/akveo/blur-admin/issues/4#issuecomment-207947902>
>



-- 
Best Regards,
Edwin Yip

**Comment by lugovsky** [comment]
@edwinyzh Unfortunately we don't have HTML files for this template as we were doing Angular version right away

**Comment by extensionsapp** [comment]
@lugovsky Да, чистая HTML версия этого шаблона была бы как нельзя кстати. Очень красивая!

**Comment by lugovsky** [comment]
@edwinyzh The problem is that raw HTML still will require a lot of javascript for navigation, initialization of widgets etc. In my eyes it's completely new project. 

But maybe we'll think about doing some "light" version with blurred panels.

Anyway I'll reopen issue to see how big interest is

**Comment by jamatos** [comment]
+1

**Comment by junaidrao** [comment]
+2

**Comment by hooloovooo** [comment]
+3

**Comment by GSWemigo** [comment]
+

**Comment by Schrims** [comment]
+

**Comment by jdpnielsen** [comment]
Please use the 'reaction' button instead of the +1 spam.

**Comment by forktheweb** [comment]
+1 do it in Aurelia-Typescript-Webpack!  yaaaaaa :)

**Comment by gadzan** [comment]
+1

**Comment by Stereotip22** [comment]
+1

**Comment by bGaetan** [comment]
+1

**Comment by Stereotip22** [comment]
No, I don't have , unfortunately

**Comment by scratcher28** [comment]
+1

**Comment by AllyW** [comment]
+1

**Comment by Polomad5** [comment]
++ It would be fantastic !

**Comment by ursuleacv** [comment]
+1

**Comment by recci** [comment]
Yes a raw html and js version would be better, I initially thought you could run gulp or grunt to build the compressed css and js files then you would not need all those dependencies once you were finished with your customisations. You cant really use this for any applcation because for the app I want to use it for it would not be a good idea to include all those bower components etc.

**Comment by rahmanQaiser** [comment]
Please release non angular version.

**Comment by lxytdsd** [comment]
+1 for non-Angular version

**Comment by honzagula** [comment]
+1

**Comment by jakir12** [comment]
Have any Row Html Version .. I need one..:)

**Comment by alihatam** [comment]
+1 , for raw html version

**Comment by Irikos** [comment]
+1, lots of people would love this and there is no other good free equivalent. I'm sure many people would consider contributing to the effort.

**Comment by DroneZzZko** [comment]
+ for vanilla.js version

**Comment by pinta83** [comment]
+1 for raw html version
The template is bitchin' :)

**Comment by sergioedo** [comment]
:+1:

**Comment by Mehran914** [comment]
+1
